.gitignore

```diff
@@ -17,5 +17,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```

Comment:
Technically this should be included in the commit but this is also a huge debate so to each their own

Reply:
I did this cause we're using yarn for the project so there's no point having both.